src/frontend/src/components/render/ModelType.tsx

```diff
@@ -0,0 +1,130 @@
+import { t } from '@lingui/macro';
+
+export enum ModelType {
+  part = 'part',
+  supplierpart = 'supplierpart',
+  manufacturerpart = 'manufacturerpart',
+  partcategory = 'partcategory',
+  stockitem = 'stockitem',
+  stocklocation = 'stocklocation',
+  build = 'build',
+  company = 'company',
+  purchaseorder = 'purchaseorder',
+  salesorder = 'salesorder',
+  salesordershipment = 'salesordershipment',
+  returnorder = 'returnorder',
+  address = 'address',
+  contact = 'contact',
+  owner = 'owner',
+  user = 'user'
+}
+
+interface ModelInformatonInterface {
+  label: string;
+  label_multiple: string;
+  url_overview: string;
+  url_detail: string;
+}
+
+type ModelDictory = {
+  [key in keyof typeof ModelType]: ModelInformatonInterface;
+};
+
+export const ModelLinks: ModelDictory = {
```

Comment:
Maybe name this more general instead of links, like you did with the interface so that we can store all meta information about a node in there if there is any more.